ui/pages/Project/selectors.js

```diff
@@ -105,16 +105,24 @@ export const getProjectAnalysisGroupFamiliesByGuid = createSelector(
 
 export const getProjectAnalysisGroupFamilyIndividualCounts = createSelector(
   getProjectAnalysisGroupFamiliesByGuid,
-  familiesByGuid => Object.values(familiesByGuid).map(family => (family.individualGuids || []).length),
+  familiesByGuid => Object.values(familiesByGuid).map(family => ({
+    size: (family.individualGuids || []).length,
+    hasMom: (family.maternalGuids || []).length === 1,
+    hasDad: (family.paternalGuids || []).length === 1,
```

Comment:
Is it possible for a family to have 3 generations and one grandma and one dad? If yes, `hasMon` and `hasDad` may not work for deciding the family structure.

Reply:
Great catch. I will update the logic, I'll need to refactor what we are actually sending back from the server in order to make this work

Reply:
Okay, previously the case you outlined here would have returned `{maternalGuids: ['grandma'], paternalGuids: ['dadd]}`. I have updated the server response to keep the association between parents so this will now come back as `{parents: [{maternalGuid: 'grandma', paternalGuid: null}, {maternalGuid: null, paternalGuid: 'dad'}]}` and then I updated the client side code to not to consider parents if there is more than one parent object returned as that means its a complex structure